master/usage/index.md

```diff
@@ -1,5 +1,101 @@
----
-title: Using Calico
----
-
-This section contains information on using Calico.
+---
+title: Using Calico
+---
+
+This section assumes you have some familiarity with Calico and have at
+least read through
+[Getting Started]({{site.baseurl}}/{{page.version}}/getting-started)
+for an orchestrator.  The information here will hopefully provide direction
+on achieving some common tasks and possibly some no so common tasks.
+
+## Common tasks
+
+#### Configuring calico/node
+
+The calico/node container is the main component in a Calico setup and as such
+there are many aspects of it that can/need to be configured.
+
+- When running calico/node on a host with multiple IP Addresses or interfaces
+  the correct one must be selected, see 
+  [Configuring a Node's IP Address and Subnet]({{site.baseurl}}/{{page.version}}/usage/configuration/node)
+  to set or change the IP Address or method used to select it.
+- Some Calico installation options may handle installing and running
+  calico/node but some deployments will want to do this manually, the page
+  [Running Calico Node Container as a Service]({{site.baseurl}}/{{page.version}}/usage/configuration/as-service)
+  helps in those cases.
+- When running Calico in a Kubernetes cluster is is possible to use
+  the Kubernetes datastore, see
+  [Configuring to use the Kubernetes Datastore Driver]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/hosted/kubernetes-datastore/#configuration-details)
+  for configuring calico/node in that situation.
+- For additional configuration refer to the 
```

Comment:
How bout: For a full reference of all flags processed / used / read (i'm not sure which is best), refer to the [calico/node configuration reference]